cliquet/views/batch.py

```diff
@@ -116,7 +116,10 @@ def build_request(original, dict_obj):
     :param original: the original batch request.
     :param dict_obj: a dict object with the sub-request specifications.
     """
+    api_prefix = '/'.join(original.upath_info.split('/')[:2])
```

Comment:
If we have path_info = "/v1/toto/tata", then the split returns `['', 'v1', 'toto', 'tata']`

and then we join it again . We should do a format with a `"/{prefix}".format(prefix=original.upath_info.split('/')[1])`